src/edit/DistortableCollection.Edit.js

```diff
@@ -320,6 +252,81 @@ L.DistortableCollection.Edit = L.Handler.extend({
     }, this);
     return this;
   },
+
+  startExport: function() {
```

Comment:
Hm, are you sure this works? I had to move it into a closure to give it access to the `this` scope.

Ah, i see, you used `bind` instead, down [on this line](https://github.com/publiclab/Leaflet.DistortableImage/pull/519/files#diff-bcc442a813f58d74ef88d4afa361a169R328)

OK!